style.css

```diff
@@ -49,6 +75,86 @@ main {
     overflow-x: hidden !important;
   }
 
+  .project_details {
+    position: fixed;
+    top: 0;
+    right: 0;
+    left: 0;
+    bottom: 0;
+    margin: auto;
+    width: 80%;
+    height: 100%;
+    background-color: #fff;
+    z-index: 10;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    visibility: hidden;
+    opacity: 0;
+    border: 1px solid #dfe1e6;
+    border-radius: 3%;
+    padding-top: 15%;
+    padding-bottom: 16%;
+    padding-left: 2%;
+    padding-right: 1%;
+  }
```

Comment:
- [ ] Kindly as the previous reviewer asked, Your mobile version still now show all the content please you can give the content smaller width so everything can appears or you can make it scrollable so the user can scroll down.

| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/35535130/156864369-63138729-5b9d-494e-9aaa-9c1cf682a3a0.png) | ![image](https://user-images.githubusercontent.com/35535130/156864382-bf6f2c0c-d300-4227-9639-fcf50ab2f209.png) |